### PR TITLE
Add ruleset: adidas.ca

### PR DESCRIPTION
**Pull request by injust**

### CODE REVIEW

**Comment by J0WI** [comment]
You may try an exclusion and a custom test url (same error on #15142).

File: src/chrome/content/rules/Adidas.ca.xml [hunk 1]
```diff
@@ -8,6 +8,9 @@
 	<target host="www.adidas.ca" />
 	<target host="cfg.adidas.ca" />
 	<target host="cp.adidas.ca" />
+		<exclusion pattern="^http://cp\.adidas\.ca" />
```
Comment (J0WI) [inline src/chrome/content/rules/Adidas.ca.xml]: Should be `^http://cp\.adidas\.ca/$`

Comment (injust) [reply src/chrome/content/rules/Adidas.ca.xml]: Oops, fixed. Thanks!

**Comment by injust** [comment]
@J0WI Looks like any URL on `cp.adidas.ca` times out during the CI build. I'll go ahead and add a whitelist entry.

**Pull request merged**